src/templates/chat/roll-treasure.html

```diff
@@ -1,26 +1,26 @@
-<div class="ose chat-card treasure-card">
-  <header class="card-header flexrow">
-    <img src="systems/ose/assets/treasure.png" title="{{table.name}}" width="36" height="36" />
-    <h3>{{table.name}}</h3>
-  </header>
-  <div class="card-content">
-    <ol class="treasure-list">
-      {{#each treasure as |t|}}
-      <li class="treasure flexrow">
-        {{#if t.img}}<img src="{{t.img}}" width="36" height="36" />{{/if}}
-        <div>
-          <div>{{{t.text}}}</div>
-          <ol>
-            {{#each t.treasure}}
-            <li class="sub flexrow">
-              {{#if img}}<img src="{{img}}" width="28" height="28" />{{/if}}
-              <div>{{{text}}}</div>
-            </li>
-            {{/each}}
-          </ol>
-        </div>
-      </li>
-      {{/each}}
-    </ol>
-  </div>
+<div class="ose chat-card treasure-card">
+  <header class="card-header flexrow">
+    <img src="{{asset '/treasure.png'}}" title="{{table.name}}" width="36" height="36" />
+    <h3>{{table.name}}</h3>
+  </header>
+  <div class="card-content">
+    <ol class="treasure-list">
+      {{#each treasure as |t|}}

```

Comment:
While we're in here... thoughts on adding an `#else` block to this `#each` for when there's no treasure, to let the user know they didn't get any treasure?

Reply:
@anthonyronda what's the general approach to getting adding new localization strings?

Reply:
Short answer is "add English strings as commits, omit all other languages"
Language contributors will get an email if they have email alerts set whenever https://crowdin.com/ autosync discovers new strings

I will add this to my list for improving documentation

Reply:
noting that "No Items" was added